sdk/tables/azure-data-tables/azure/data/tables/aio/_policies_async.py

```diff
@@ -112,7 +155,7 @@ async def send(self, request):  # pylint: disable =W0236
                 break
             except AzureError as err:
                 retries_remaining = self.increment(
-                    retry_settings, request=request.http_request, error=err)
+                    retry_settings, error=err)
```

Comment:
just a question: why do you no longer need to pass request?

Reply:
`request` isn't used in `self.increment` at all, pylint noticed that for me